test/models.test.js

```diff
@@ -50,10 +52,74 @@ describe('collections', function () {
 
 describe('models', function() {
   describe('centre', function() {
+    beforeEach(function() {
+      var io;
+
+      this.server = new socket();
+      this.fakeio = function(server) {
+        return function() {
+          return server.socketClient;
+        }
+      };
+
+      io = sailsIOClient(this.fakeio(this.server));
+      io.sails.autoConnect = false;
+
+      this.socket0 = io.sails.connect();
+    });
     it('should be instantiated', function() {
       var model = new models.Centre();
 
       expect(model).not.to.be(undefined);
+      expect(model.get('name')).to.be(undefined);
+      expect(model.get('centre_id')).to.be(undefined);
+      expect(model.get('beds')).to.eql([]);
+      expect(model.get('booked')).to.be(0);
+      expect(model.get('reserved')).to.be(0);
+
+    });
+
+    it('should respond to socket events', function(done) {
+      var model = new models.Centre([], { socket: this.socket0 });
+
+      model.socket.on('message', function(payload) {
+        expect(payload).to.be('test message');
+        done();
```

Comment:
ideally do this with promises rather than passing done which will allow you to ultimately return the expect back up to mocha easily